src/hs_dbus_signature/_signature.py

```diff
@@ -12,6 +12,7 @@
 _CODES = ("b", "d", "g", "h", "i", "n", "o", "q", "s", "t", "u", "v", "x", "y")
 
 
+# pylint: disable=too-many-arguments
 def dbus_signatures(
```

Comment:
Please move the ```disable``` line onto the line below. The ```disable``` above the function call disables that lint for everything below it in the file, including all additional functions in the file (in this case, this is the only function, but that's unusual).

```def dbus_signatures(   #pylint...```